src/Skills.jsx

```diff
@@ -125,11 +125,10 @@ class Skills extends Component {
 
   componentDidMount() {
     const currentId = (this.props.params) ? this.props.params.id : null;
-    if (!currentId) {
+    if (!currentId)
```

Comment:
Bad habit, explicit curly braces! http://stackoverflow.com/questions/4797286/are-curly-braces-necessary-in-one-line-statements-in-javascript